src/Reflection/ReflectionClass.php

```diff
@@ -688,6 +689,34 @@ public function isInstance($object)
     }
 
     /**
+     * Checks whether the given object is an instance
```

Comment:
I think it would be worth clarifying exactly the difference between this and `isInstanceOf`